front/app/containers/ProjectsShowPage/shared/header/ProjectActionButtons.tsx

```diff
@@ -51,193 +55,163 @@ interface Props {
 const ProjectActionButtons = memo<Props>(({ projectId, className }) => {
   const project = useProject({ projectId });
   const phases = usePhases(projectId);
-
   const [currentPhase, setCurrentPhase] = useState<IPhaseData | null>(null);
-  const [ideasPresentOutsideViewport, setIdeasPresentOutsideViewport] =
-    useState(false);
-  const [surveyPresentOutsideViewport, setSurveyPresentOutsideViewport] =
-    useState(false);
-  const [pollPresentOutsideViewport, setPollPresentOutsideViewport] =
-    useState(false);
+  const divId = clHistory.location.hash;
```

Comment:
Let's use regular search query parameters instead of hashes to implement this - it will make it more consistent with other places in the app.

Reply:
I agree that's generally true but in the context of our app, we are not using hashes anywhere, as far as I know. Just trying to keep it consistent and not introduce new ways of doing things, unless really necessary 👍

Reply:
After some reading and thought, I think this is one the cases where using the hash will be better for us. Semantically and for maintaining the code. The query string doesn't explicitly tell what is happening. One needs to read more code where us a hash does that. Browsers also support hashes by default and that is always a good fall back to using any JS logic. 

https://developer.mozilla.org/en-US/docs/Web/API/URL/hash
https://developer.mozilla.org/en-US/docs/Web/API/URL/href#Examples

Reply:
Alright, in this case, I agree there's a good argument to be made for using a hash. Let's go with that, if you have a strong preference 👍 

But let's access it with the router hook `useLocation`, instead of directly from `clHistory`.